Automattic/wp-calypso
# Hosting Command Palette: Add minor UI changes to command palette styles

- Related to https://github.com/Automattic/dotcom-forge/issues/4531

## Proposed Changes

This PR affects the UI of the command palette in the following ways:

* it removes the horizontal scroll on the site items for the nested commands:

<img width="2530" alt="Screenshot 2023-11-23 at 2 30 07 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/3cbedc92-055c-408e-8adb-1a9b5fda2fbb">

* for the sites that do not have a title, we are adding the display of the domain instead - this is an approach that is similar to the site switcher:

<img width="2526" alt="Screenshot 2023-11-23 at 2 32 06 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/fd0cdb1c-1b3c-462b-9565-5ea4948c3845">

* it splits the site title and site domain into two lines as opposed to being on one line as before:

<img width="437" alt="Screenshot 2023-11-23 at 2 34 03 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/2e197cf5-4b3d-4612-819a-bf74c8143eec">

## Testing Instructions

* Navigate to `/sites` with Calypso Live Link
* Press `cmd + k` on Mac and `ctrl + k` on Windows
* Observe that the changes described above are present 

## Pre-merge Checklist
